### PR TITLE
issue: Language Pack Locale Mismatch

### PR DESCRIPTION
**Pull request by JediKev**
This addresses issue #4325 where the Language Pack Locales are mismatched.
The Locale for the first language is displayed on the second language,
etc. This updates the `$manifest` variable to be set before we display
data so the correct `MANIFEST` file is included and all language data is
displayed correctly.

### CODE REVIEW

**Comment by qatuno** [comment]
So, i do know nothing. But would like to test and patch my fork for building up and run a new machine. Somehow i'm new into github. Just some system integrater who is always up and can't sleep.  In the end i'm not 100% sure if it is about the problem i have on to my running installation.

**Comment by JediKev** [comment]
@qatuno

Unfortunately I do not understand your question. Could you rephrase it for me?

Cheers.

**Comment by qatuno** [comment]
Yes sure.  How do i use this patch? Is it already usable out of the main repository. Or do i really need to fork this into my own repository. Sorry me, Please don't get me wrong. Even if it sounds stupid.

**Comment by JediKev** [comment]
@qatuno

It does not sound stupid 🙂

If your current installation is a download from http://osticket.com/download the best way is to make the changes manually by editing your current installation’s codebase. If your current installation is a clone of the repo then you can pull this pull request locally and merge it to your codebase. If you can’t do any of the above then you can wait until it is approved/merged and released in the next version.

Cheers.

**Comment by qatuno** [comment]
sounds good. told you i can't sleep and always looking for something interesting. this is an experience to me, like to learn. I'm some autodidact. This and all the cases which i like to get into. 
If you like to know. I'm very much into OsTicket. It is an fine solution for some one who worked into service and had seen many plattforms and needed to work with. Nice straight forward.

all the best
